src/components/AddressSearch.js

```diff
@@ -137,7 +151,7 @@ const AddressSearch = (props) => {
             values.street = street;
         }
 
-        if (_.includes(CONST.ALL_COUNTRIES, country)) {
+        if (CONST.ALL_COUNTRIES[country]) {
```

Comment:
This could cause a crash in android if the country name is missing on the ALL_COUNTRIES

Reply:
May be could use lodashGet